src/config/appSettings.ts

```diff
@@ -0,0 +1,28 @@
+import { UUID } from '@graasp/sdk';
+
+import Agent from '@/types/Agent';
+
+type AssistantSettings = Omit<Agent, 'type'>;
+
+export type AssistantsSettingsType = {
+  assistantsList: AssistantSettings[];
```

Comment:
Very minor: usually convention is you either use singular + list (e.g., `assistantList`) or just plural (`assistants`).

Reply:
changed everywhere:
exchangesList => exchangeList
assistantsList => assistantList
PropTypesList => PropTypeList